frontend/src/services/LendService.tsx

```diff
@@ -1,9 +1,38 @@
+import { ItemType } from "../Types/types";
 import BackendClient from "./HttpService";
+import axios from "axios";
 
 
-export async function getLendItems(){
-	let response = await BackendClient.get("/api/item")
-	return response
-}
+const lendService = {
+	async getItems(){
+		let response = await BackendClient.get("/api/item")
+		return response
+	},
+
+	async createItem(item: ItemType) {
+		let response = await BackendClient.post(
+			"/api/item", 
+			{
+				name: item.name, 
+				owner_id: item.id, 
+				lend_start: item.lend_start, 
+				lend_end: item.lend_end, 
+				img_uri: item.img_uri
+			}
+		)
+		return response
+	},
 
+	async isValidImageUri(imguri: string){
+		let response = await axios.get(imguri)
```

Comment:
This causes a CORS error. Permissions need to be changed, or it can be an optional field with alternate text if link is broken.